secboot/encrypt_sb.go

```diff
@@ -66,6 +74,25 @@ func AddRecoveryKey(key keys.EncryptionKey, rkey keys.RecoveryKey, node string)
 	return keymgr.AddRecoveryKeyToLUKSDeviceUsingKey(rkey, key, node)
 }
 
+func runSnapFDEKeymgr(args []string, stdin io.Reader) error {
+	toolPath, err := snapdtool.InternalToolPath("snap-fde-keymgr")
+	if err != nil {
+		return fmt.Errorf("cannot find keymgr tool: %v", err)
```

Comment:
I forgot, we should probably be consistent calling this "FDE key manager tool" in errors/messages